### ISSUE TITLE
Transition altitude 1000 feet higher than supposed to be when retrieving from Simbrief? 

### ISSUE THREAD

**Issue by xhemp** [flybywiresim/a32nx#6269]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-11-29T01:13:37+00:00",
  "ref": "HEAD",
  "sha": "6803283d85f97decbccf60eb3162d701ec71b217",
  "actor": "Andreas Guther <andreas@guther.family>",
  "event_name": "manual"
}
```

### Describe the bug

After commit https://github.com/flybywiresim/a32nx/commit/c066df3f7db74ea994b11c4b571a6128528eda5d , it looks to me the transition altitude retrieved from simbrief is always 1000 feet higher than what is in the charts. Wondering if this is me looking at the wrong place or if it's something else playing up.

### Expected behavior

Have the same transition altitude as per the trans alt on APP/SID charts.

### Steps to reproduce

1. Import flight plan from SB
2. Check the transition altitude matches the one from charts

### Discord Username (optional)

XH#7761

**Comment by MPO910** [comment]
I do see this also! Constantly reproduceable when importing flightplans from simbrief

**Comment by tracernz** [comment]
Do you have any examples? The data comes direct from SimBrief, and for them I assume directly from navdata.

**Comment by xhemp** [comment]
Sure thing, let me do get a quick flight going and I will get some screenshots but again, I can be looking at the wrong place :)

**Comment by MPO910** [comment]
Yes. For example LEBL, LPPR, LPPD, EDDF....and many more ....

Marcus
________________________________

**Comment by xhemp** [comment]
Apologies for the large images!

SBFL - SBGL

Transition for SID (BODO1B) is correct, 9000
![2021-11-29 (2)](https://user-images.githubusercontent.com/13650956/143942593-ca32213f-2c30-4a39-bd42-03c9611a43f6.png)
![2021-11-29 (5)](https://user-images.githubusercontent.com/13650956/143942864-b5ce8d35-ecf5-49f5-a3c2-539e203e950c.png)

Transition for runway 15 is 8000 but MCDU shows 9000:
![2021-11-29 (3)](https://user-images.githubusercontent.com/13650956/143942763-5bc0c25a-cc51-4d4d-8b95-b3aea14ff655.png)
![2021-11-29 (7)](https://user-images.githubusercontent.com/13650956/143943198-0bc2c4aa-9696-4cc8-baa2-8caf34aa6ec3.png)

I don't remember which ones were but I've seen both being incorrect as well and every single time the difference was 1000 feet.
Am I looking at the wrong place?

**Comment by tracernz** [comment]
You should be looking at transition _level_ for the approach page... it's "By ATC" which I guess has been replaced with a typical value by SimBrief (i.e. 1000 foot transition layer). Ideally we wouldn't fill the approach page in this case, but we have no way to tell, and outside Europe transition levels are coded.

**Comment by xhemp** [comment]
I think this explained the whole thing very well: https://mediawiki.ivao.aero/index.php?title=Transition_altitude_and_transition_flight_level
Thanks for checking this, @tracernz and for chiming in @MPO910

**Comment by MPO910** [comment]
Yes am aware of this. But why importing something that isn't accurate (as normally given by ATC) ? Maybe it's better to do import on the approach Value?

That would avoid many questions, and users can search at the charts or put in a value "as if" they pretend to be ATC-once given values.

Marcus
________________________________

**Comment by tracernz** [comment]
It doesn't help that the Honeywell MCDU labels it TRANS ALT instead of TRANS FL... the latest Thales MCDU has it right.

> Yes am aware of this. But why importing something that isn't accurate (as normally given by ATC) ? Maybe it's better to do import on the approach Value?

That would be wrong for the rest of the world, where transition level is a hard value. The value given in Europe is at least a reasonable one (trans lvl = trans alt + 1000). I think this is the right tradeoff, and a perfect one for most of the world.

**Comment by tracernz** [comment]
Duplicate of https://github.com/flybywiresim/a32nx/issues/2068 really.
